README.md

```diff
@@ -111,6 +111,7 @@ for (const el of document.querySelectorAll('[data-shortcut]')) {
    3. Due to the inconsistent lowercasing of `event.key` on Mac and iOS when `Meta` is pressed along with `Shift`, it is recommended to avoid hotkey strings containing both `Mod` and `Shift`.
 7. `"Plus"` and `"Space"` are special key names to represent the `+` and ` ` keys respectively, because these symbols cannot be represented in the normal hotkey string syntax.
 8. You can use the comma key `,` as a hotkey, e.g. `a,,` would activate if the user typed `a` or `,`. `Control+,,x` would activate for `Control+,` or `x`.
+9. `"Shift"` should be included if it would be held, and the key name should match the 'uppercase' key name: `Shift+?`, `Shift+A`, `Shift+$`
```

Comment:
We should alter this explanation as I don't think it holds true.

Reply:
Which part of it? It should be correct if this PR works as expected.

Reply:
we do have point `#3` above:

> 3. Due to the inconsistent lowercasing of `event.key` on Mac and iOS when `Meta` is pressed along with `Shift`, it is recommended to avoid hotkey strings containing both `Mod` and `Shift`.

but I think what Keith is saying is this isbn't true on a Mac when the Cmd key is held.

Reply:
Updated to 
> 9. `"Shift"` should be included if it would be held and the key is uppercase: ie, `Shift+A` not `a`. Note however that MacOS outputs lowercase keys when `Command+Shift` is held; see 6.3 above.